src/main/kotlin/no/nav/familie/tilbake/dokumentbestilling/manuell/brevmottaker/ManuellBrevmottakerRepository.kt

```diff
@@ -0,0 +1,15 @@
+package no.nav.familie.tilbake.dokumentbestilling.manuell.brevmottaker
+
+import no.nav.familie.tilbake.common.repository.InsertUpdateRepository
+import no.nav.familie.tilbake.common.repository.RepositoryInterface
+import no.nav.familie.tilbake.dokumentbestilling.manuell.brevmottaker.domene.ManuellBrevmottaker
+import org.springframework.stereotype.Repository
+import java.util.UUID
+
+@Repository
+interface ManuellBrevmottakerRepository :
```

Comment:
Kan være fint med en integrasjonstest som legger inn 1 (og 2) brevmottakere og henter opp de, og sjekker at objektene er like 

Tenker det også kan være fint å ha en test for egne metoder i disse jdbc-interface, i dette tilfelle `findByBehandlingId`